packages/now-cli/src/util/deploy/process-deployment.ts

```diff
@@ -37,6 +38,10 @@ export default async function processDeployment({
           `Total files ${event.payload.total.size}, ${event.payload.missing.length} changed`
         );
 
+        if (!quiet) {
+          log(`Synced ${event.payload.missing.length} ${uploadStamp()}`);
```

Comment:
```suggestion
          log(`Synced ${pluralize(event.payload.missing.length, 'file', true)} ${uploadStamp()}`);
```

(`pluralize()` might need to be imported)